src/js/views/patients/schedule/schedule_views.js

```diff
@@ -98,11 +98,11 @@ const DayItemView = View.extend({
     <td class="schedule-list__action-list-cell schedule-list__patient">
       <div class="schedule-list__state-patient">
         <span class="schedule-list__action-state action--{{ stateOptions.color }}">{{fa stateOptions.iconType stateOptions.icon}}</span><span class="schedule-list__search-helper">{{ state }}</span>&#8203;{{~ remove_whitespace ~}}
-        <span class="schedule-list__patient-name js-patient">{{ patient.first_name }} {{ patient.last_name }}</span>&#8203;
+        <span class="schedule-list__patient-name {{#unless isReduced}}js-patient{{/unless}} {{#if isReduced}}is-reduced{{/if}}">{{ patient.first_name }} {{ patient.last_name }}</span>&#8203;
```

Comment:
this could probably be `{[#if isReduced}}...{{else}}...{{/if}}`

Reply:
I think this class swapping is fine.  However an alternative.. which i don't have a strong opinion on is:
```js
triggers() {
  if (this.isReduced) {
    return {
       'click .js-form': 'click:form',
    };
  }

  return {
    'click .js-form': 'click:form',
    'click .js-patient': 'click:patient',
    'click': 'click',
    'click .js-select': 'click:select',
  };
}
```

Reply:
Hmm that's an interesting alternative way to do it... I'll play around with both and see which one I like better.